Intersect.Editor/Forms/Editors/frmResource.cs

```diff
@@ -19,29 +18,15 @@ namespace Intersect.Editor.Forms.Editors;
 
 public partial class FrmResource : EditorForm
 {
-
-    private List<ResourceDescriptor> mChanged = [];
-
-    private string mCopiedItem;
-
-    private ResourceDescriptor mEditorItem;
-
-    private Bitmap mEndBitmap;
-
-    private Bitmap mEndGraphic;
-
-    private Bitmap mInitialBitmap;
-
-    private Bitmap mInitialGraphic;
-
-    private List<string> mKnownFolders = [];
-
-    private bool mMouseDown;
-
-    private BindingList<NotifiableDrop> _dropList = [];
-
-    //General Editting Variables
-    bool mTMouseDown;
+    private readonly Dictionary<Guid, ResourceDescriptor> _changed = [];
+    private string? _copiedItem;
+    private ResourceDescriptor? _editorItem;
+    private Bitmap? _graphicBitmap;
+    private Bitmap? _resourceGraphic;
+    private bool _mouseDown;
+    private readonly List<string> _knownFolders = [];
+    private readonly BindingList<NotifiableDrop> _dropList = [];
+    private readonly BindingList<ResourceStateDescriptor> _stateList = [];
```

Comment:
Can you reorder this so all of the `readonly`s are together at the top with one empty line between them and the mutable fields?